controller/user_controller.rb

```diff
@@ -25,6 +25,20 @@ def title
 			end
 		end
 
+		post '/update_hashtag' do
+			settings.log.info(params)
+			block_unsigned
+			return if params[:id].nil? || params[:value].nil? || !check_if_valide_hashtag?(params[:value])
+			id_hashtag = Hashtag.where(equality: {name: params[:value]})[0].id
+			if (current_user.is_related_with(id: id_hashtag, type_of_link: "APPRECIATE") == [])
+				current_user.create_links(id: id_hashtag, type: "APPRECIATE", data: nil)
+			else
+				current_user.suppress_his_relation_with(id: id_hashtag)
```

Comment:
maybe I should do a toggle function

Reply:
There is only two place where we would use such function